src/vendor/azure-uamqp-c/deps/azure-c-shared-utility/pal/ios-osx/tlsio_appleios.c

```diff
@@ -521,7 +521,14 @@ static void dowork_poll_socket(TLS_IO_INSTANCE* tls_io_instance)
     CFStreamCreatePairWithSocketToHost(NULL, tls_io_instance->hostname, tls_io_instance->port, &tls_io_instance->sockRead, &tls_io_instance->sockWrite);
     if (tls_io_instance->sockRead != NULL && tls_io_instance->sockWrite != NULL)
     {
-        if (CFReadStreamSetProperty(tls_io_instance->sockRead, kCFStreamPropertySSLSettings, kCFStreamSocketSecurityLevelNegotiatedSSL))
+
+        CFStringRef keys[1] = {kCFStreamPropertySocketSecurityLevel};
+        CFStringRef values[1] = {kCFStreamSocketSecurityLevelNegotiatedSSL};
```

Comment:
Do these need to be cleaned up at all?

Reply:
I don't think it's needed, my understanding is it's a reference to the system defined constants, the code doesn't do the dynamic memory allocation (copy/create/etc.).